pom.xml

```diff
@@ -302,7 +302,7 @@
 		<dependency>
 			<groupId>orca</groupId>
 			<artifactId>ndl</artifactId>
-			<version>5.1.5-SNAPSHOT</version>
+			<version>5.2.2-SNAPSHOT</version>
```

Comment:
This isn't necessarily wrong in this particular case, but by default / in general, Flukes should be using Release artifacts from Orca.

Reply:
I'll fix that